tools/proofers/preview.js

```diff
@@ -495,6 +495,8 @@ var makePreview = function (txt, viewMode, styler) {
                 return match;
             }
             noNote = removeComments(p1);
+            // remove tags so that all uppercase string is correctly identified
+            noNote = noNote.replace(/&lt;\/?.&gt;/g, '');
```

Comment:
It should be sufficient to match single letter tags because the string is inside <sc> tags so another <sc> tag (or a <tb>) would cause another error before we got here.

Reply:
Works for me, thanks!